### ISSUE TITLE
[Bug] Enabling check Hit in Workflow makes all hits critical

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#762]
In GitLab by @anarion321 on Mar 14, 2022, 03:29

I'm using Foundry VTT Version 9, build 255. The only Game system I got installed it's "Dnd5e - Fifth Edition System 1.5.7", I deactivated all modules but Midi QOL and it's 2 dependencies, I've got latest version of all, libWrapper 1.12.4.0, Midi QOL 0.9.32 and socketlib 1.0.10

When I enable Hits in workflow, all hits are shown as critical hits, if i do not enable it, it works fine.

![Capture](https://my-gitlab-bucket.s3.amazonaws.com/640711674/f5bee6db03de4c9d969b709c3ff97aab94ae7c005a4d17ec54ad268f2bdd960a/Capture.PNG)

**Comment by tposney** [comment]
Can you make sure you are on 0.9.33? I pushed that out specially to fix that problem.

**Comment by tposney** [comment]
In GitLab by @anarion321 on Mar 14, 2022, 19:43

Thanks for the quick response!!
I just tried and seems to work fine with that version :smile: 

It no longer says critical hit to every hit, it detects hits and misess. I did not get a 20 or a 1 in a few throws of the dice so i cannot tell you if critical works, but seems fixed.

I'm not an expert on gitlab, but I'll close the Issue.

![Capture](https://my-gitlab-bucket.s3.amazonaws.com/640711674/35b24f42725e443a10904ace92f381f2c142ba20e83a41314c813107dc0471ee/Capture.PNG)
